Book/label.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_reader :id, :title, :color, :items
```

Comment:
Over here if you use attr_accessor method for title, color, and items it would be better, so later on you can change the attributes. In the current case, you can only use the getter method for the attributes.